src/GraphQL.Tests/Execution/AstFromValueTests.cs

```diff
@@ -12,22 +12,22 @@ public class AstFromValueTests
         public void converts_null_to_null()
         {
             object value = null;
-            var result = value.AstFromValue(null, null);
+            var result = ((IGraphType)null).ToAST(value);
             result.ShouldBeOfType<NullValue>();
         }
 
         [Fact]
         public void converts_string_to_string_value()
         {
-            var result = "test".AstFromValue(null, new StringGraphType());
+            var result = new StringGraphType().ToAST("test");
             result.ShouldNotBeNull();
             result.ShouldBeOfType<StringValue>();
         }
 
         [Fact]
         public void converts_bool_to_boolean_value()
         {
-            var result = true.AstFromValue(null, new BooleanGraphType());
+            var result = new BooleanGraphType().ToAST(true);
             result.ShouldNotBeNull();
             result.ShouldBeOfType<BooleanValue>();
```

Comment:
```suggestion
            result.ShouldBeOfType<BooleanValue>().Value.ShouldBe(true);
```

and so on for other tests